src/metabase/types.clj

```diff
@@ -89,6 +90,7 @@
 
 (derive :type/CreationTimestamp :type/DateTime)
 (derive :type/JoinTimestamp :type/DateTime)
+(derive :type/Birthday :type/DateTime)
```

Comment:
Would it be better to derive birthday from `:type/Date` since we won't have a time (I assume). Also :+1: birthday->birthdate to fit the date rhyme scheme